app/config/config.yml

```diff
@@ -30,7 +30,9 @@ framework:
         save_path:   "%kernel.root_dir%/../var/sessions/%kernel.environment%"
     fragments:       ~
     http_method_override: true
-    assets: ~
+    assets:
+        # set version of assets to simplify cache flushing
+        version: v=1
```

Comment:
Add version of assets to config. This will help us to flush client-side cache for assets